terra/reports.py

```diff
@@ -624,38 +620,6 @@ def get_individual_data_employee(employee_ids, start_date=None, end_date=None):
         .values("profdev_expend")
     )
 
-    admin_alloc = (
-        TravelRequest.objects.filter(
-            traveler=OuterRef("pk"),
-            administrative=True,
-            closed=False,
-            departure_date__gte=start_date,
-            return_date__lte=end_date,
-        )
-        .values("traveler__pk")
-        .annotate(admin_alloc=Sum("funding__amount"))
-        .values("admin_alloc")
-    )
-
-    admin_expend = (
-        TravelRequest.objects.filter(
-            traveler=OuterRef("pk"),
-            administrative=True,
-            closed=True,
-            departure_date__gte=start_date,
-            return_date__lte=end_date,
-        )
-        .values("traveler__pk")
-        .annotate(admin_expend=Sum("actualexpense__total"))
-        .values("admin_expend")
-    )
-
-    days_vacation = TravelRequest.objects.filter(
```

Comment:
@aprigge TRRA-179 doesn't say anything about removing Vacation days (done here and at other places in code below).  Is this removal correct?

Reply:
@akohler. Yes, we weren't displaying the vacation days in the employee report so I removed that (though the ticket doesn't explicitly mention it).